src/main/java/com/spikes2212/command/drivetrains/commands/DriveArcade.java

```diff
@@ -7,14 +7,15 @@
 
 
 /**
- * this command moves a {@link TankDrivetrain} by linear and rotational speeds, using
+ * This command moves a {@link TankDrivetrain} by linear and rotational speeds, using
```

Comment:
Change it to `A commands that moves...` according to B-I\a2 : `The documentation of a class should describe its instance, instead of the class itself`